test_scaffold/utils.py

```diff
@@ -31,6 +33,10 @@ def create_test_user(
         email=email,
     )
 
+    if should_associate_user_email:
+        user_email = associate_user_email(user, email, confirmed=True)
+        user_email.set_primary_email()
```

Comment:
The `set_primary_email()` call is not needed.

This sets `user.email = user_email.email`; the former already has this value.

See: https://github.com/hacktoolkit/django-htk/blob/dc48e5f972805b1e0c6fe8f72e3b86b3fd862fbb/apps/accounts/models.py#L253-L272

Reply:
Got it. Thanks for providing this explanation, I see it now. I need to be careful when moving code over.